### PR TITLE
[Installer] Support WasmEdge rustls plugin installation

### PR DESCRIPTION
**Pull request by hydai**

### CODE REVIEW

**Comment by juntao** [comment]
Hello, I am a [code review bot](https://github.com/flows-network/github-pr-summary/) on [flows.network](https://flows.network/).

It could take a few minutes for me to analyze this PR. Relax, grab a cup of coffee and check back later. Thanks!

File: utils/install.py [hunk 1]
```diff
@@ -387,6 +389,10 @@ def compare(self, version2, separator=". |-", ignorecase=True):
     "manylinux2014" + "x86_64" + WASMEDGE_IMAGE_PLUGIN: VersionString("0.13.0"),
     "manylinux2014" + "aarch64" + WASMEDGE_IMAGE_PLUGIN: VersionString("0.13.0"),
     "ubuntu20.04" + "x86_64" + WASMEDGE_IMAGE_PLUGIN: VersionString("0.13.0"),
+    "darwin" + "x86_64" + WASMEDGE_RUSTLS: VersionString("0.13.4"),
```
Comment (SAtacker) [inline utils/install.py:392]: I don't see a release for this, is there?

File: utils/install.py [hunk 2]
```diff
@@ -387,6 +389,10 @@ def compare(self, version2, separator=". |-", ignorecase=True):
...
+    "darwin" + "arm64" + WASMEDGE_RUSTLS: VersionString("0.13.4"),
+    "manylinux2014" + "x86_64" + WASMEDGE_RUSTLS: VersionString("0.13.4"),
+    "ubuntu20.04" + "x86_64" + WASMEDGE_RUSTLS: VersionString("0.13.4"),
```
Comment (SAtacker) [inline utils/install.py:395]: Could you please add a test cases for this ?

File: utils/install.py (hunk 1 above)
Comment (hydai) [reply utils/install.py:392]: Here: https://github.com/WasmEdge/WasmEdge/releases/tag/0.13.4

File: utils/install.py (hunk 2 above)
Comment (hydai) [reply utils/install.py:395]: Sure. Will process it.

File: utils/install.py (hunk 1 above)
Comment (SAtacker) [reply utils/install.py:392]: Ohh, for 0.13.5 it isn't there

Comment (SAtacker) [reply utils/install.py:392]: https://github.com/WasmEdge/WasmEdge/releases/download/0.13.5/WasmEdge-plugin-wasmedge_rustls-0.13.5-darwin_x86_64.tar.gz

Comment (hydai) [reply utils/install.py:392]: Ah... The reason is that the macOS-11 CI runner is broken. So, all artifacts are not built and updated. We are still fixing this issue. If it's working again, then the release assets will be updated soon.

**Comment by hydai** [comment]
The macOS-lastest will fail as expected Because the related 0.13.5 assets are not built.

**Comment by SAtacker** [comment]
Just noting here the url so we don't have to read the whole log again https://github.com/WasmEdge/WasmEdge/releases/download/0.13.5/WasmEdge-plugin-wasmedge_tensorflow-0.13.5-darwin_x86_64.tar.gz

**Comment by juntao** [comment]
Can we merge?

**Review by SAtacker — APPROVED** [review]

**Pull request merged**